frontend/src/modules/Environments/components/EnvironmentTeams.js

```diff
@@ -215,9 +193,34 @@ function TeamRow({ team, environment, fetchItems }) {
                 }
               />
             </LoadingButton>
-          )}
-        </Box>
-      </TableCell>
+            <LoadingButton
+              loading={loadingCreds}
+              onClick={() => generateCredentials(team.groupUri)}
+            >
+              <CopyAllOutlined
+                sx={{
+                  color:
+                    theme.palette.mode === 'dark'
+                      ? theme.palette.primary.contrastText
+                      : theme.palette.primary.main
+                }}
+              />
+            </LoadingButton>
+            {team.groupUri !== environment.SamlGroupName && (
```

Comment:
Would this also remove the ability the `removeGroup` from the environment? Should this be loaded regardless of `isFeatureEnabled('core', 'env_aws_actions')`?

Reply:
@noah-paige apologies! did not realize this was happening. Fixed!